projects/netgrif-components-core/src/lib/side-menu/content-components/user-assign/base-user-assign-list/abstract-base-user-assign-list.component.ts

```diff
@@ -0,0 +1,62 @@
+import {Component, Input, OnDestroy, OnInit, ViewChild} from "@angular/core";
+import {RolesObject, UserListItem, UserListService} from "../../../../user/services/user-list.service";
+import {FormControl} from "@angular/forms";
+import {CdkVirtualScrollViewport} from "@angular/cdk/scrolling";
+import {debounceTime} from "rxjs/operators";
+import {UserValue} from "../../../../data-fields/user-field/models/user-value";
+
+@Component({
+    selector: 'ncc-base-user-assign-list',
+    template: ''
+})
+export abstract class AbstractBaseUserAssignListComponent implements OnInit, OnDestroy {
+
+    protected SEARCH_DEBOUNCE_TIME = 600;
```

Comment:
didn't we refactor debounce time constants, or am I just imagining things?

Reply:
I do not remember such a refactor. It was originally in `AbstractUserAssignListComponent` for a long time, but needed to create another component for `AbstractMultiUserAssignListComponent` and needed similar functions, so I made an abstraction above these two and copied the duplicable code.